windows.c

```diff
@@ -97,7 +101,7 @@ int initialize(INTERNET_PER_CONN_OPTION_LIST* options) {
 int query(INTERNET_PER_CONN_OPTION_LIST* options) {
   DWORD dwBufferSize = sizeof(INTERNET_PER_CONN_OPTION_LIST);
   if(!InternetQueryOption(NULL, INTERNET_OPTION_PER_CONNECTION_OPTION, options, &dwBufferSize)) {
-    reportWindowsError("Querying options");
+    reportWindowsError("querying options", options->pszConnection ? options->pszConnection : "LAN");
```

Comment:
Ok, I ended up use ternary op instead as the compiler complains when I use `options->pszConnection || "LAN"`.
```
windows.c:104:44: warning: passing argument 2 of 'reportWindowsError' makes pointer from   integer without a cast [-Wint-conversion]
     reportWindowsError("querying options", options->pszConnection || "LAN");
                                            ^
windows.c:9:6: note: expected 'const char *' but argument is of type 'int'
 void reportWindowsError(const char* action, const char* connName) {
      ^
```